BitPlanner/Scenes/CraftPage.cs

```diff
@@ -132,6 +133,61 @@ public void CopyTreesAsText()
         DisplayServer.ClipboardSet(result.ToString().Trim());
     }
 
+    public void CopyTreesAsCSV()
+	{
+	var result = new StringBuilder();
+	var separator = new string('-', 64);
+
+	result.AppendLine("Item Name,Generic Name,Profession/Skill,Quantity,In Stock");
+
+		foreach (var tab in _recipeTabs.GetChildren().Cast<RecipeTab>())
+		{
+		var root = tab.GetTreeRoot();
+		TraverseAndAppendCSV(root, result);
+		result.AppendLine(separator); // Separera olika träd
+		}
+
+	DisplayServer.ClipboardSet(result.ToString().Trim());
+	}
+
+	private void TraverseAndAppendCSV(TreeItem item, StringBuilder csv)
+	{
+		var id = item.GetMetadata(0).AsUInt64();
+		var craftingItem = _data.CraftingItems[id];
+
+		// Get quantity
+		var quantity = item.GetMetadata(2).AsInt32Array();
+		var minQuantity = quantity[0];
+		var maxQuantity = quantity[1];
+
+		var quantityString = RecipeTab.GetQuantityString((uint)minQuantity, maxQuantity >= 0 ? (uint)maxQuantity : 0u);
+
+		// Get profession/skill
+		int skill = -1;
+		if (craftingItem.ExtractionSkill > -1)
+		{
+			skill = craftingItem.ExtractionSkill;
+		}
+		else if (craftingItem.Recipes.Count > 0)
+		{
+			skill = craftingItem.Recipes[0].LevelRequirements[0];
+		}
+
+		var skillName = Skill.GetName(skill);
+
+		// Format: Item Name,Generic Name,Profession/Skill,Quantity,In Stock
+		var itemName = craftingItem.Tier > -1 ? $"{craftingItem.Name} (T{craftingItem.Tier})" : craftingItem.Name;
+		var genericName = craftingItem.Tier > -1 ? $"{craftingItem.GenericName} — T{craftingItem.Tier}" : craftingItem.GenericName;
+
+		csv.AppendLine($"{itemName},{genericName},{skillName},{quantityString.Replace(",", "")},0");
```

Comment:
This format is useful for base ingredients list, but absolutely terrible for trees. All columns in base ingredients list CSV are supposed not only to provide info, but to also sort items however the user likes. But trees are not lists, they have fixed hierarchy that should never change.

A proper format would be similar to when copying trees as text:

```
Item,Minimum Quantity,Maximum Quantity,In Stock
Name (Tx),minq,maxq,0
| Name (Tx),minq,maxq,0
  | Name,minq,maxq,0
```